File: pgm/ABC_Java/ABC_386_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        int A, B, C, D;
        A = Integer.parseInt (arr[0]);
        B = Integer.parseInt (arr[1]);
        C = Integer.parseInt (arr[2]);
        D = Integer.parseInt (arr[3]);
        int[] cards = new int[13];
        cards[A - 1]++;
        cards[B - 1]++;
        cards[C - 1]++;
        cards[D - 1]++;
        int threeCards = 0;
        int pairs = 0;
        for (int i = 0; i < cards.length; i++) {
            if (cards[i] == 3) {
                threeCards++;
                break;
            } else if (cards[i] == 2) {
                pairs++;
            }
        }
        System.out.println (threeCards == 1 ? Yes : pairs == 2 ? Yes : No);
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_386_A.vb
' Option Strict On
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim ABCD() As Integer = Array.ConvertAll(
        Console.ReadLine().Split(),
        Function(x As String) Integer.Parse(x))
        Const thirteen As Integer = 13
        Const Yes As String = "Yes"
        Const No As String = "No"
        Dim A As Integer = ABCD(0)
        Dim B As Integer = ABCD(1)
        Dim C As Integer = ABCD(2)
        Dim D As Integer = ABCD(3)
        Dim cards(thirteen) As Integer
        cards(A - 1) += 1
        cards(B - 1) += 1
        cards(C - 1) += 1
        cards(D - 1) += 1
        Dim threeCards As Integer = 0
        Dim pairs As Integer = 0
        For i As Integer = 0 To thirteen - 1
            If cards(i) = 3 Then
                threeCards += 1
                Exit For
            ElseIf cards(i) = 2 Then
                pairs += 1
            End If
        Next
        If threeCards = 1 OrElse pairs = 2 Then
            Console.WriteLine(Yes)
        Else
            Console.WriteLine(No)
        End If
        Return
    End Sub
End Class